ai_edge_torch/generative/examples/stable_diffusion/README.md

```diff
@@ -2,13 +2,48 @@
 This example shows how to use the Edge Generative API to convert a PyTorch Stable Diffusion v1.5 model to TFLite model, and run the image generation inference.
 
 ## Convert PyTorch to TFLite model
+The example provides two source checkpoints mapping. One is original HuggingFace repo, and the other one is a third party PyTorch implementation of Stable Diffusion. Users need to specify the source checkpoints mapping format for conversion script.
+
+### SafeTensors model checkpoints from original HuggingFace repo
+1. Clone original HuggingFace stable diffusion repo [runwayml/stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5)
+2. Run `convert_to_tflite.py` and use `v1-5-pruned-emaonly.safetensors` as the source checkpoints for the conversion script. Notice that optional encoder model is not supported yet.
+```bash
+python ai_edge_torch/generative/examples/stable_diffusion/convert_to_tflite.py \
+--clip_ckpt=$HOME/stable-diffusion-v1-5/v1-5-pruned-emaonly.safetensors \
+--diffusion_ckpt=$HOME/stable-diffusion-v1-5/v1-5-pruned-emaonly.safetensors \
+--decoder_ckpt=$HOME/stable-diffusion-v1-5/v1-5-pruned-emaonly.safetensors \
+--output_dir=/tmp/stable_diffusion_safetensors/ \
+--ckpt_format=safetensors
+```
+
+
+### PyTorch model checkpoints from third party
```

Comment:
let's just remove this and only support safetensors format.

Reply:
Done